lib/jekyll-sitemap.rb

```diff
@@ -59,7 +59,7 @@ def sitemap_content
       site_map.content = File.read(source_path)
       site_map.data["layout"] = nil
       site_map.render({}, @site.site_payload)
-      site_map.output.gsub(/\s*\n+/, "\n")
+      site_map.output.gsub(/\s{2,}/, "\n")
```

Comment:
Actually this is not equivalent. As `\s` contains not only `\n` but also `\t` and space, the whole `" \n\t"` or `" \n "` will be replaced, while `/\s*\n+/` replaces only `" \n"`. As a result, the indentation of `sitemap.xml` has disappeared. Is this intended?